fpm/src/fpm/cmd/new.f90

```diff
@@ -137,34 +267,91 @@ subroutine cmd_new(settings)
         call warnwrite(join_path(settings%name, 'app/main.f90'), littlefile)
     endif
 
+    if(settings%with_full)then
+       tomlfile=[character(len=80) ::  tomlfile,   &
+       &'[dependencies]                                                                  ', &
+       &'#                                                                               ', &
+       &'#Files will be searched for automatically (by default) in                       ', &
+       &'# src/, app/, test/, and example/.                                              ', &
```

Comment:
This explanation might be better placed near the `auto-executables` etc. entries.